src/Tasks/Microsoft.NET.Build.Tasks/PackageInfo.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml;
+using System.Globalization;
+using System.Xml.Linq;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    internal class PackageInfo : IEquatable<PackageInfo>
```

Comment:
I think we should use [https://github.com/NuGet/NuGet.Client/blob/4cccb13833ad29d6a0bcff055460d964f1b49cfe/src/NuGet.Core/NuGet.Packaging.Core.Types/PackageIdentity.cs#L13](https://github.com/NuGet/NuGet.Client/blob/4cccb13833ad29d6a0bcff055460d964f1b49cfe/src/NuGet.Core/NuGet.Packaging.Core.Types/PackageIdentity.cs#L13) instead of making our own class.  PackageIdentiy from NuGet already has all the correct semantics built into it.